### DISCUSSION TITLE
用reality协议时，如果GFW对vps的ip主动探测，它会看到什么？

### DISCUSSION THREAD

**Discussion by kashiwa-eco** [XTLS/Xray-core#2414] [Q&A]
根据 https://github.com/XTLS/Xray-core/discussions/2200#discussioncomment-6149065, 
我了解到reality本身就能放主动探测。不过我自己用浏览器主动访问搭建了reality节点的vps的IP，结果是什么内容也没有（这是理所当然的）。那么如果GFW也主动访问vps的IP，自然也是看到一个没有任何内容的网站。那么我长时间访问一个没有内容的IP，会被GFW认为是异常行为吗？

**Comment by chika0801** [comment]
你看这里 https://github.com/XTLS/REALITY 最后一段的描述

`REALITY 客户端应当收到由“临时认证密钥”签发的“临时可信证书”，但以下三种情况会收到目标网站的真证书：`

如果你要在客户端测试REALITY转发的效果，方法是修改本机hosts文件，内容为 vps ip 空格 你在dest处填的域名，保存后。关，开浏览器，输入你dest的域名，显示的内容为dest域名网站的内容。

**Reply by kashiwa-eco** [reply]
非常感谢回复！很遗憾以本人对于计算机的粗浅知识，不少内容还无法理解。目前我就搭个节点先用着吧，反正这段时间用起来也挺稳定的。期待reality加入xray正式版内核后能有更详细的文档

**Comment by chika0801** [comment]
> 那么我长时间访问一个没有内容的IP，会被GFW认为是异常行为吗？

这问题见链接 ： https://github.com/XTLS/Xray-core/issues/2358#issuecomment-1646622927

我建议是：你可以把你的VPS，给你几个朋友用，这样就是只有你们几个人长时间访问你的VPS了。
